pkg/action/cl001/ac001/executor.go

```diff
@@ -2,8 +2,68 @@ package ac001
 
 import (
 	"context"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/infrastructure/v1alpha2"
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/awscnfm/pkg/key"
+	"github.com/giantswarm/awscnfm/pkg/release"
 )
 
 func (e *Executor) execute(ctx context.Context) error {
+	var err error
+
+	var releaseComponents map[string]string
+	{
+		c := release.Config{}
+
+		releaseCollection, err := release.New(c)
+		if err != nil {
+			return microerror.Mask(err)
+		}
+
+		releaseComponents = releaseCollection.ReleaseComponents(key.Release)
+	}
+
+	var crs v1alpha2.ClusterCRs
+	{
+		c := v1alpha2.ClusterCRsConfig{
+			ClusterID:         "cl001",
+			Domain:            key.DomainFromHost(e.clients.ControlPlane.RESTConfig().Host),
+			Description:       "awscnfm cluster cl001",
+			Owner:             "giantswarm",
+			Region:            key.RegionFromHost(e.clients.ControlPlane.RESTConfig().Host),
+			ReleaseComponents: releaseComponents,
+			ReleaseVersion:    key.Release,
+		}
```

Comment:
This is probably the most standard way of creating a Tenant Cluster now so all defaults will be applied. Further cluster scopes can define that how they please.